contracts/VoucherHub.sol

```diff
@@ -188,7 +191,10 @@ contract VoucherHub is
         VoucherHubStorage storage $ = _getVoucherHubStorage();
         require($._isVoucher[voucher], "VoucherHub: unknown voucher");
         _mint(voucher, value); // VCHR
-        IERC20($._iexecPoco).transfer(voucher, value); // SRLC
+        if (!IERC20($._iexecPoco).transfer(voucher, value)) {
+            // SRLC
+            revert("VoucherHub: SRLC transfer to voucher failed");
+        }
```

Comment:
Same question here.

Reply:
We can use a common function (something like `transfertFundsToVoucher(address, value)`) if it does not consume more gas.

Reply:
+11 gas wich is fine I add it [here](https://github.com/iExecBlockchainComputing/iexec-voucher-contracts/pull/26/commits/bc24e63085f4e6a3705da855a07f28e029410fc4)